reviewer_config.prototxt

```diff
@@ -0,0 +1,93 @@
+display_name: "StartupOS"
+repo {
+  id: "startup-os"
+  url: "https://github.com/google/startup-os"
+}
+repo {
+  id: "hasadna"
+  url: "https://github.com/hasadna/hasadna"
+}
+project {
+  id: "google_oss_examples"
+  display_name: "Google Open Source examples"
+  project_url: "https://github.com/google/startup-os"
+  description: "Working examples of Google's open source stack: Bazel, Protocol Buffers, gRPC, Angular, Dagger, Docker, Kubernetes etc."
+  user_id: "oferb"
+}
+project {
+  id: "reviewer"
+  display_name: "Reviewer: Multi-repo Code Review"
+  project_url: "https://github.com/google/startup-os/tree/master/tools/reviewer"
+  description: "A multi-repo Code Review tool that is hosting-agnostic and supports atomic commits across repos."
+  user_id: "oferb"
+}
+user {
+  id: "oferb"
+  project_id: "google_oss_examples"
+  project_id: "reviewer"
+  top_contribution {
+    description: "Team leader for StartupOS project (Google OSS examples, Reviewer)"
+    url: "https://github.com/google/startup-os"
+  }
+  top_contribution {
```

Comment:
I'd remove all `top contribution` for all users exclude you. They will edit the file and add actual data by your own.

Reply:
We can fix it later. For now it will help with testing.